src/app/modules/issue-vc/new-issue-vc/new-issue-vc.component.ts

```diff
@@ -198,9 +103,9 @@ export class NewIssueVcComponent implements OnInit {
     };
 
     return {
-      fields: this.fieldList,
-      claimType: this.selectedNamespace,
-      claimTypeVersion: parseVersion(this.selectedRole.version) || DEFAULT_CLAIM_TYPE_VERSION
+      claimType: this.getFormType().value.namespace,
+      claimTypeVersion: parseVersion(this.selectedRoleDefinition.version) || DEFAULT_CLAIM_TYPE_VERSION,
```

Comment:
Is this working? I am thinking if this.selectedRoleDefinition.version is missing, then parseVersion might return error. Not sure. Might be good to do a ternary operator here:
`this.selectedRoleDefinition.version ? parseVersion(this.selectedRoleDefinition.version) : DEFAULT_CLAIM_TYPE_VERSION,`

Reply:
As far as i know, version always exist. This is kinda copy paste from different place [Look here](https://github.com/energywebfoundation/switchboard-dapp/blob/382bd85e2a3e2d0c347db0648636cfd6f71196ee/src/app/routes/registration/request-claim/request-claim.component.ts#L316-L332). This is another compromise - to not change/adjust different part of code where there is no tests. I will add TODO to change this in the future :)